app/views/pictures/index.html.haml

```diff
@@ -0,0 +1,29 @@
+- if @pictures.any?
+  - @pictures.reverse.each do |picture|
+    %div.img_container
+      %ul.img_info
+        %li.username= picture.user.username
+        %li.timestamp= picture.timestamp
+      = image_tag picture.image.url(:medium)
+      %div.caption_container
+        %h4= picture.caption
+        - if current_user && current_user.id == picture.user_id
+          = link_to image_tag('/images/edit.png', :class => "edit"), edit_picture_path(picture), class: 'edit-link'
+          = link_to image_tag('/images/delete.png', :class => "delete"), picture_path(picture), class: 'delete-link', method: :delete
+      %p
+        %span.likes_count= picture.likes.count
+        = 'like'.pluralize picture.likes.count
+      - if picture.comments.any?
+        %ul
+          - picture.comments.each do |comment|
+            %li= "#{comment.user.username} #{comment.thoughts}"
+      %div.interact_container
```

Comment:
`%div.interact_container` can be written as `.interact_container` since `%div` is implicit